server/src/main/java/org/opensearch/transport/CompressibleBytesOutputStream.java

```diff
@@ -58,13 +58,13 @@
  *
  * @opensearch.internal
  */
-final class CompressibleBytesOutputStream extends StreamOutput {
+public final class CompressibleBytesOutputStream extends StreamOutput {
```

Comment:
This class could be kept package private (unchanged) and moved to `nativeprotocol` package: it is used only by native protocol